test-integration/scores.coffee

```diff
@@ -43,13 +49,16 @@ describe 'CC Student Scores', ->
     @scores.goCCScores()
 
   @it 'sorts by name or data', ->
-    @scores.el.nameHeaderSort.get().click() # If you use just `.click()` that will scroll to try to make the element visible but cause the next element to not be clickable
-    @scores.el.dataHeaderSort.click()
-    @user.goHome() # Because the logout from a CC page goes to cc.openstax, go home first so the logout goes to tutor.
+    @scores.el.scoreCell.isPresent().then (isPresent) =>
+      return console.log('Skipping because there are no assignments') unless isPresent
+
+      @scores.el.nameHeaderSort.click()
+      @scores.el.dataHeaderSort.click()
+      @user.goHome() # Because the logout from a CC page goes to cc.openstax, go home first so the logout goes to tutor.
 
   @it 'changes periods', ->
     @scores.el.periodTab.click()
-    @user.goHome() # Because the logout from a CC page goes to cc.openstax, go home first so the logout goes to tutor.
```

Comment:
because the same comment was repeated from the goHome() above it.  I can add it back if you want.